### PR TITLE
structure commit

### PR DESCRIPTION
**Pull request by nithina**
Daily status page structure.

### CODE REVIEW

**Comment by mad-nishin** [comment]
@nithina 
1. Use a generic folder structure that includes controllers/ services etc.
2. Rename main.js to app.js
3. Do not club controller and servies with app file.

File: index.html [hunk 1]
```diff
@@ -1 +1,42 @@
-My Page
+<!DOCTYPE html>
+<html>
+    <head>
```
Comment (mad-nishin) [inline index.html:3]: Our app has 2 views. So remove the status view from index and add it to a partial. and then load the partial using routesindex

**Comment by mad-nishin** [comment]
@nithina ,
Added feedback. Please update the changes and revert..

**Comment by nithina** [comment]
@nishintn done

**Pull request merged**